Create header section for mobile features

Armynerh opened the pull request:
Add and linters
Add and modify index.html
Add and modify style.css
add and re-modify README.md
re-modify style.css to pass linter errors

Armynerh committed:
create linters and modify README.md

Armynerh committed:
modify linters

Armynerh committed:
add index.html

Armynerh committed:
add style.css

Armynerh committed:
add image folder

Armynerh committed:
modify style.css

Armynerh committed:
re-modify style.css

Armynerh committed:
re-modify style.css

Armynerh committed:
re-modify style.css again

Armynerh committed:
re-modify style.css for linter checks

Armynerh committed:
add new images to the folder

Armynerh committed:
modify index.html

Armynerh committed:
modify style.css for mobile features

Armynerh committed:
re-modify style.css for mobile features

Armynerh committed:
re-modify style.css for mobile features

Armynerh committed:
edit README.md

Armynerh committed:
edit README.md

souad988 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+        </nav>
+    </div>
+    </header>
+    <div class="main-container">
```
great work on wrapping the main work in a div but it's better to use a main tag as container as you might have many div tag inside the same section this will make you code clear organized and clean

souad988 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+        <div id="main-item1">
+            <h2>Hey there. <span id="main-span1">I'm Kenny</span>. <span id="main-span2">I'm a software developer</span></h2>
+        </div>
+        <div id="main-item2">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
```
here a p tag is more adequate for displaying text always use it for your paragraph for semantic reasons

souad988 commented on index.html:
```diff
@@ -0,0 +1,38 @@
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </div>
+        <div id="main-item3">
+            <img src="images/Social-media.png">
+        </div>
```
social media icons are used as links to redirect the user to your accounts kindly use each of them as icon wraped in an anchor a tag with a valid href link to your account and since you have a list of them please make sure to put them in ul tag 
exp `<ul><li><a href="" ><img .../></a></li></ul>`

souad988 reviewed (changes requested):
Hi @Armynerh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Good use of github workflows
No linter errors
Good PR title

## Required Changes ♻️
_Check the comments under the review._
_Also kindly consider adjusting the posisioning of the background image make it no repeat and align the humberger icon more to the right .
and reduce font size of the second title to be like the design otherwise you did great keep up the good work 

![image (6)](https://github.com/Armynerh/personal_portfolio/assets/59707859/7ffbba89-8851-49e1-bc75-a320ead330a4)
![image (7)](https://github.com/Armynerh/personal_portfolio/assets/59707859/1d1c5bd9-b9c4-459e-a1d4-2d69e3364b0e)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Armynerh committed:
modify index.html to include main tag

Bantealem reviewed (changes requested):
# Status: Required Changes ♻️♻️♻️ 
Hi @Armynerh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] No Linters Error
- [x] Correct Github flow
- [x] Professional README 

 ## Required Changes ♻️
- [ ] Before you request the second review please make sure you have made changes to the request by the first reviewer. Currently, you didn't implement any changes requested by the first reviewer. Kindly, implement these required changes and ask for a review. 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Armynerh commented:
@Bantealem Thank you for your review. Could you please point out what further changes need to be implemented?
I have wrapped the div element with a main tag, I have included a p tag  for the text file, and replaced the social Icons with clickable images. Outside that, I have also fixed the positioning of the background image and made it no repeat, lastly I also fixed the toggle icon. Those were the recommended changes, could you please point out specifically what you think I might have missed? Thank you :)

Armynerh commented on index.html:
```diff
@@ -20,19 +20,26 @@
     </div>
     </header>
-    <div class="main-container">
+    <main class="main-container">
```
@Bantealem Here are some of the said changes.

Armynerh committed:
re-modify style.css for mobile features

Armynerh committed:
modify index.html to include main tag

Armynerh committed:
modify background image

Armynerh committed:
modify index.html to include main tag

emmiiorji reviewed (approved):
Hi @Armynerh, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>
<img src='https://user-images.githubusercontent.com/20554583/220871050-9893fe6a-46a0-42a3-8a44-46ac0f7f1154.gif'/>
</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
